.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+<!--
+Use this template as a guide. Omit sections that don't apply.
+More details about the template are at https://github.com/edx/open-edx-proposals/pull/180
+-->
+
+## Description
+
+Describe what this pull request changes, and why. Include implications for people using this change.
```

Comment:
Also, is this a good opportunity to [link to OEP-19](https://open-edx-proposals.readthedocs.io/en/latest/oep-0019-bp-developer-documentation.html#adrs), which discusses what should go in READMEs versus ADRs versus comments.

Reply:
I added a link per @nedbat's suggestion.